installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc

```diff
@@ -52,8 +52,6 @@ include::modules/installation-requirements-user-infra.adoc[leveloffset=+1]
 * See link:https://www.ibm.com/support/knowledgecenter/en/SSB27U_7.1.0/com.ibm.zvm.v710.hcpa6/bhslzvs.htm[Bridging a HiperSockets LAN with a z/VM Virtual Switch] in the IBM Knowledge Center.
```

Comment:
In line 22 could you change
s/. Prerequisites/== Prerequisites

Reply:
@SNiemann15 thank you for your review and suggestions. I have now resolved this suggestion.